pkgs/servers/monitoring/loki/default.nix

```diff
@@ -14,7 +14,12 @@ buildGoModule rec {
 
   vendorSha256 = null;
 
-  subPackages = [ "..." ];
+  subPackages = [
+    "cmd/loki"
+    "cmd/loki-canary"
+    "cmd/promtail"
+    "cmd/logcli"
```

Comment:
While at it, can we split promtail, loki and logcli into different packages? I don't need promtail and logcli on every machine reporting logs, its a 1:n relationship for most deployments. And logcli is also useful on machines not running loki themselves, but querying them.

Reply:
I put a TODO above. I won't get sonish to do the split myself so it will be either merged with it or the PR stalls until I come back or someone else has implemented the change. Your call.

Reply:
Different packages or outputs?

Reply:
Different packages would be less surprising for users I guess because they can be overwritten and are more visible.